guides/common/modules/con_secure-boot.adoc

```diff
@@ -0,0 +1,19 @@
+[id="secure-boot"]
+= Secure boot
```

Comment:
```suggestion
= Secure Boot
```
I think this should get an exception for capitalization since it's a standard feature name.

Reply:
Applied and added to the list of exceptions in a third commit.